CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 ### Fixes
 
+- Bring protocol up to date with latest Sentry protocol ([#1038](https://github.com/getsentry/sentry-dart/pull/1038))
```

Comment:
We have to move to unreleased section

Reply:
Fixed it. You're doing way too many releases lately, I can barely keep up with it :D